pkg/config/singleflight.go

```diff
@@ -0,0 +1,15 @@
+package config
+
+// SingleFlight holds the various
+// backend configurations for a distributed
+// lock or single flight mechanism.
+type SingleFlight struct {
+	Etcd *Etcd
+}
+
+// Etcd holds client side configuration
+// that helps Athens connect to the
+// Etcd backends.
+type Etcd struct {
+	Endpoints string
```

Comment:
Can you explicitly name this and the above struct field with tags? I think explicit is better than implicit here

Reply:
@arschles +1 on this field, missed the env override. But the above struct field does not need it just like the `StorageConfig`